feature/sessions/README.md

```diff
@@ -76,24 +76,24 @@ install(Sessions) {
 
 **Notes about security:**
 
-* Serving session without transforms allows people to see in the clear the contents of the session and to modify it
-* Serving session with an Authentication transform allows people to see the contents, but prevents modifying it as long
-  as you keep your secret hash key safe and use a secure algorithm. Also allows to use old session strings to go back
+* Serving a session without transform allows people to see the contents of the session clearly and to modify it
+* Serving a session with an Authentication transform people can see the contents, but prevents them from modifying it as long
+  as you keep your secret hash key safe and use a secure algorithm. It is also possible to use old session strings to go back
   to a previous state.
-* Serving session with an Encrypt transform prevents from actual determining the contents and preventing modification,
-  but still is vulnerable to going back to previous states.
+* Serving a session with an Encrypt transform prevents people from determining the actual contents and modification it,
+  but it is still vulnerable when going back to previous states.
   
-It is possible to store a timestamp or a nonce encrypted and authenticated, but you would have to time limit the
-session or to verify it at the server, reducing the benefits from this mode.
+It is possible to store a timestamp or a nonce encryption and authentication, but you will have to limit the
```

Comment:
Wanted to say that the "timestamp or that nonce" is encrypted and authenticated. (it is not a method of encryption or authentication)